bundle/bundle.go

```diff
@@ -25,7 +26,7 @@ const (
 	RegoExt     = ".rego"
 	JSONExt     = ".json"
 	ManifestExt = ".manifest"
-	DataFileExt = "/data.json"
+	DataFile    = "data.json"
```

Comment:
[nit] Since this file seems to be the only one affected by the rename here, this could probably be not exported?

Reply:
Makes sense. Will fix.